calabash-cucumber/lib/calabash-cucumber/store/preferences.rb

```diff
@@ -40,10 +107,8 @@ def ensure_preferences_dir
       def defaults
         {
           :version => VERSION,
-          :usage_tracking =>
-          {
-
-          }
+          :usage_tracking => "system_info",
+          :user_id => SecureRandom.uuid
```

Comment:
A new :user_id is generated when the :user_id key is empty or missing in the .json preferences file.

It is stable across loads _until it disappears_.  It will only disappear if the user removes the key from the .json preferences file or the preferences file itself is removed.